beacon-chain/blockchain/forkchoice/lmd_ghost_yaml_test.go

```diff
@@ -134,5 +134,7 @@ func TestGetHeadFromYaml(t *testing.T) {
 		}
 
 		helpers.ClearAllCaches()
+		testDB.TeardownDB(t, db)
```

Comment:
above, at the beginning of the cycle, a `defer testDB.TeardownDB(t, db)` is defined

Reply:
we need to teardown the db for each iteration, the defer will only teardown at the end of function call

Reply:
I was afraid that if you call TeardownDB twice, it will return an error. But db.Close () checks that the database is open. And os.RemoveAll does not return an error if the folder does not exist. Ok, double call of TeardownDB is normal.

Reply:
Normally we shouldnt double call it, but with the prometheus collector being registered whenever we instantiate a DB, it returns an error if we register it another time without deregistering it first. So thats why I add the teardown there, to prevent the test failure